app/helpers/application_helper/toolbar/middleware_deployment_center.rb

```diff
@@ -15,4 +16,26 @@ class ApplicationHelper::Toolbar::MiddlewareDeploymentCenter < ApplicationHelper
       ]
     ),
   ])
+  button_group('middleware_deployments_operations', [
+    select(
+      :middleware_deployment_deploy_choice,
+      'fa fa-play-circle-o fa-lg',
+      t = N_('Operations'),
+      t,
+      :items => [
+        button(
+          :middleware_deployment_redeploy,
+          'pficon pficon-restart fa-lg',
+          N_('Redeploy this #{ui_lookup(:table=>"middleware_deployment")}'),
```

Comment:
So this will not work properly (the translation). But we will fix it with @mzazrivec over all of the toolbars once he gets back from PTO.